includes/discovery/ports/slms.inc.php

```diff
@@ -0,0 +1,18 @@
+<?php
```

Comment:
I'd prefer to avoid yet another custom port polling implementation if we could.  Can you just set bad_if on the OS instead?